pkgs/development/compilers/zasm/default.nix

```diff
@@ -23,22 +25,23 @@ stdenv.mkDerivation {
     ln -sf ${libs-src} Libraries
   '';
 
-  buildPhase = ''
-    cd Linux
-    make
-  '';
+  makeFlags = [
+    "CC=${stdenv.cc.targetPrefix}cc"
+    "CXX=${stdenv.cc.targetPrefix}c++"
+    "LINK=${stdenv.cc.targetPrefix}c++"
+    "STRIP=${stdenv.cc.targetPrefix}strip"
+  ];
 
   installPhase = ''
-    mkdir -p $out/bin
-    mv zasm $out/bin
+    install -Dm755 -t $out/bin zasm
   '';
 
   meta = with lib; {
-    description = "Z80 / 8080 assembler (for unix-style OS)";
+    description = "Z80 / 8080 / Z180 assembler (for unix-style OS)";
     homepage = "https://k1.spdns.de/Develop/Projects/zasm/Distributions/";
     license = licenses.bsd2;
     maintainers = [ maintainers.turbomack ];
-    platforms = platforms.linux;
+    platforms = platforms.unix;
     badPlatforms = platforms.aarch64;
```

Comment:
if you want to try aarch please remove the badPlatform from here and we can try it using OfBorg build